src/EditorFeatures/CSharpTest2/Recommendations/DecimalKeywordRecommenderTests.cs

```diff
@@ -732,5 +732,14 @@ void Method()
     }
 }");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestInFunctionPointerType()
+        {
+            await VerifyKeywordAsync(@"
+class C
+{
+    delegate*<$$");
```

Comment:
would like at least one test of:
1. not getting these after `delegate*`
2. getting thse after `delegate*<int, `
3. getting these after `delegate*<ref ` (and after comma as well).

this doesn't have tobe duplciated into each file. but we should have it in at least one of them.

Reply:
i honestly would prefer not to. i don't get 'test fatigue' personally.  the test sits there enert, doing it's duty in a trivial to understand fashion.

more complex test harnesses annoy the heck out of me as, so often,  cannot actually tell what they're doing and if they're doing it properly.